x/mint/abci.go

```diff
@@ -20,15 +29,22 @@ func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
 	// recalculate inflation rate
 	totalStakingSupply := k.StakingTokenSupply(ctx)
 	bondedRatio := k.BondedRatio(ctx)
-	minter.Inflation = minter.NextInflationRate(params, bondedRatio)
```

Comment:
Is there any way that we can make this configurable?  Constructing the `mint` module with an inflation calculator or similar, for instance.

This would make it easier to get included in the upstream repository.

Reply:
We will try to look on it.

Reply:
We will try to add some function to mind module which takes function as input. Default will be the current SDK mint calculation logic. So that, we can send our custom calculation function to it from app.go or something. We will try to work on this.

Reply:
@boz , we added custom function to mint module and integrated decay formula to simapp (just for reference). Please have a glance on this: https://github.com/ovrclk/cosmos-sdk/pull/112